easybuild/easyblocks/b/blender.py

```diff
@@ -0,0 +1,97 @@
+##
+# Copyright 2009-2018 Ghent University
+#
+# This file is part of EasyBuild,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# https://github.com/easybuilders/easybuild
+#
+# EasyBuild is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation v2.
+#
+# EasyBuild is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with EasyBuild.  If not, see <http://www.gnu.org/licenses/>.
+##
+"""
+EasyBuild support for Blender, implemented as an easyblock
+
+@author: Stijn De Weirdt (Ghent University)
+@author: Dries Verdegem (Ghent University)
+@author: Kenneth Hoste (Ghent University)
+@author: Pieter De Baets (Ghent University)
+@author: Jens Timmerman (Ghent University)
+@author: Ward Poelmans (Ghent University)
+@author: Samuel Moors (Vrije Universiteit Brussel)
+"""
+import glob
+import os
+
+from easybuild.easyblocks.generic.cmakemake import CMakeMake
+from easybuild.tools.modules import get_software_root, get_software_version
+from easybuild.tools.systemtools import get_shared_lib_ext
+
+
+class EB_Blender(CMakeMake):
+    """Support for building Blender."""
+
+    def configure_step(self):
+        """Set CMake options for Blender"""
+        self.cfg['separate_build_dir'] = True
+
+        self.cfg.update('configopts', '-DWITH_INSTALL_PORTABLE=OFF')
+        self.cfg.update('configopts', '-DWITH_BUILDINFO=OFF')
+
+        # disable SSE detection to give EasyBuild full control over optimization compiler flags being used
+        self.cfg.update('configopts', '-DWITH_CPU_SSE=OFF')
+        self.cfg.update('configopts', '-DCMAKE_C_FLAGS_RELEASE="-DNDEBUG"')
+        self.cfg.update('configopts', '-DCMAKE_CXX_FLAGS_RELEASE="-DNDEBUG"')
+
+        # these are needed until extra dependencies are added for them to work
+        self.cfg.update('configopts', '-DWITH_GAMEENGINE=OFF')
+        self.cfg.update('configopts', '-DWITH_SYSTEM_GLEW=OFF')
+
+        # Python paths
+        python_root = get_software_root('Python')
```

Comment:
@smoors Make sure `Python` is actually included as a dependency (and if not, skip this configuration part):

```python
python_root = get_software_root('Python')
if python_root:
    pyshortver = '.'.join(get_software_version('Python').split('.')[:2])
    ...